src/components/Menus/VerticalMenu/VerticalTabs.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+import { Tabs } from '@/components/Tabs';
+import { useActiveTabStore } from '@/stores/activeTab';
+import { useMediaQuery } from '@mui/material';
+import type { Theme } from '@mui/material/styles';
+import { useTheme } from '@mui/material/styles';
+import { useVerticalTabs } from './useVerticalTabs';
+
+interface VerticalTabsProps {
+  verticalPageReload?: boolean;
+}
+
+export const VerticalTabs = ({ verticalPageReload }: VerticalTabsProps) => {
+  const theme = useTheme();
+  const { activeTab, setActiveTab } = useActiveTabStore();
+  const isDesktop = useMediaQuery((theme: Theme) => theme.breakpoints.up('md'));
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setActiveTab(newValue);
+  };
+  const verticalTabs = useVerticalTabs({ verticalPageReload });
+
+  const containerStyles = {
```

Comment:
should be in its own style component

Reply:
We have a few places with such logic of containerStyles which then require some cleanup if we find a better solution for this: 
![image](https://github.com/user-attachments/assets/1327fb0d-9464-4592-90c3-6f43e14d9d46)